caab-api/open-api-specification.yml

```diff
@@ -348,7 +447,7 @@ paths:
           description: 'Internal server error'
     post:
       tags:
-        - applications
+        - linked cases
```

Comment:
this is still application scoped. Should it not be tagged as application still?

Reply:
updated to both